backend/src/tasks/dnstwist.ts

```diff
@@ -4,11 +4,59 @@ import { CommandOptions } from './ecs-client';
 import { plainToClass } from 'class-transformer';
 import saveVulnerabilitiesToDb from './helpers/saveVulnerabilitiesToDb';
 import { spawnSync } from 'child_process';
+import { Client } from 'pg';
+
+function connectToPeDatabase() {
+  // connect to the PE Database.
+  const client = new Client({
```

Comment:
@aloftus23 I thought the original idea was that we would just use the existing python scripts to connect to the PE database? Will we still do that as well, in addition to the approach taken in this PR?